msg.go

```diff
@@ -860,7 +860,7 @@ func (dns *Msg) unpack(dh Header, msg []byte, off int) (err error) {
 	// The header counts might have been wrong so we need to update it
 	dh.Nscount = uint16(len(dns.Ns))
 	if err == nil {
-		dns.Extra, off, err = unpackRRslice(int(dh.Arcount), msg, off)
+		dns.Extra, _, err = unpackRRslice(int(dh.Arcount), msg, off)
```

Comment:
I'm a bit torn on this. Yea it's not read, but it breaks symmetry with the line above.

Reply:
Yeah I think back out this change and then LGTM.

Reply:
`msg.go:863:3: this value of off is never used (SA4006)`